deploy/metrics-apiservice.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: apiregistration.k8s.io/v1beta1
+kind: APIService
+metadata:
+  name: v1alpha1.metrics
+spec:
+  insecureSkipTLSVerify: true
+  group: metrics
```

Comment:
"first class" vs non-"first class" is not precisely the issue here (also, IIRC, some of the built-in versions are getting fully-qualified as well).

We should eventually move to `resource-metrics.metrics.k8s.io` or something of the sort (as opposed to `custom-metrics.metrics.k8s.io`, which is the CM group), but probably not until we move to beta, or something like that.